src/utils/getFileTypeFromPath.ts

```diff
@@ -0,0 +1,20 @@
+import { FileType } from "../tree/types";
+
+export const getFileTypeFromPath = (path: string): FileType => {
```

Comment:
needs JSDOC

Reply:
`/**
 * @function
 * @param {string} path
 * @returns FileType}
 */`

Does this look good?

Reply:
Where is the description? Push the new code and i will approve if it good or leave another comment

Reply:
should we use @description like Manpreet did, or can we put the description in front of the @function as well?

Reply:
You should remove function and description and just write the description :)

Reply:
@ralph-dev remove the JSDoc description and just write a general comment destruction?

Reply:
this part should clearly explain the function itself. Outputs the file type of a certain path. You don't need to repeat it twice.